tests/bdd/features/volume/create.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Volume creation
+
+  Background:
+    Given a control plane, Mayastor instances and a pool
```

Comment:
does this mean a single pool in the cluster?

Reply:
Yes.